.github/actions/verify-accelerated-query/action.yml

```diff
@@ -0,0 +1,77 @@
+name: 'Verify Accelerated Query'
+description: 'Verifies that an accelerated query returns the same results as a non-accelerated query'
+inputs:
+  dataset-source:
+    description: 'Definition of source dataset'
+    required: true
+  table-name:
+    description: 'Table to verify accelerated schema'
+    required: true
+  engine:
+    description: 'Acceleration engine for verification'
+    required: true
+outputs:
+  query-result:
+    description: "Query result encoded as base64 string"
+    value: ${{ steps.query.outputs.query-result }}
+  accelerated-query-result:
+    description: "Accelerated query result encoded as base64 string"
+    value: ${{ steps.query-accelerated.outputs.query-result }}
+runs:
+  using: "composite"
+  steps:
+    - name: Create spicepod
+      shell: bash
+      run: |
+        killall spice || true
+        [ -f spice.log ] && rm spice.log
+        spice init app_${{ inputs.table-name }}
+        # copy local databases (if any) to create spicepod folder
+        cp *.db app_${{ inputs.table-name }}
+
+    - name: Create non-accelerated dataset
+      shell: bash
+      working-directory: app_${{ inputs.table-name }}
+      run: |
+        echo "Creating non-accelerated dataset"
+        echo "datasets:" >> spicepod.yaml
```

Comment:
This doesn't need to change for this PR - but I'm not a huge fan of constructing spicepods manually via string manipulation. It is brittle - if a breaking change is made to the spicepod definition, its hard to find all of these places that would need to be updated. Ideally we could use the CLI or some other tool that manipulates spicepods using our existing spicepod parser.

Let's file an issue to track.

Reply:
Yeah, created the following issue: https://github.com/spiceai/spiceai/issues/1226